lm_eval/models/openai_completions.py

```diff
@@ -470,39 +505,32 @@ def sameuntil_chunks(xs, size):
                 contexts, all_gen_kwargs = zip(*chunk)
                 inps = [{"role": "user", "content": context} for context in contexts]
 
-                gen_kwargs = all_gen_kwargs[0]
+                context_gen_kwargs = all_gen_kwargs[0]
```

Comment:
didn't want there to be a confusing variable clash here with the class **gen_kwargs

Reply:
resolved since we're not using gen_kwargs anymore